crates/runtime/src/embeddings/connector.rs

```diff
@@ -98,17 +97,6 @@ impl DataConnector for EmbeddingConnector {
         }
     }
 
-    async fn stream_provider(
```

Comment:
Why did we remove this?

Reply:
It wasn't used, and this changes mode supersedes it.